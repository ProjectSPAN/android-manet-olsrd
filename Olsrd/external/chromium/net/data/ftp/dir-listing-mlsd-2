type=file;size=839;modify=19980908010757;UNIX.mode=0644;unique=6ag5b4e47d; ftpmotd
