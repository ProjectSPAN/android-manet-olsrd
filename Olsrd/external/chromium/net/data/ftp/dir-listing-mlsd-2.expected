-
ftpmotd
839
1998
9
8
1
7
